seed/migrations/0177_compliance_metric.py

```diff
@@ -17,25 +17,22 @@ class Migration(migrations.Migration):
             fields=[
                 ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
                 ('name', models.CharField(max_length=255)),
-                ('start', models.DateTimeField()),
-                ('end', models.DateTimeField()),
+                ('start', models.DateTimeField(blank=True, null=True)),
```

Comment:
should users be able to enter program compliance metrics that have no start and end dates? I feel like that will cause issues on the insights pages...